qubit-sdk/src/main/java/com/qubit/android/sdk/internal/placement/connector/PlacementConnectorBuilderImpl.kt

```diff
@@ -0,0 +1,21 @@
+package com.qubit.android.sdk.internal.placement.connector
+
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+class PlacementConnectorBuilderImpl : PlacementConnectorBuilder {
```

Comment:
I feel, creating interface and class for building PlacementConnector is over-engineering.

PlacementInteractorImpl can depend directly on PlacementConnector, not on builder of it, because PlacementConnector is constant. It doesn't change during app lifecycle. Isn't it?

Personally, I would move initialization of placementAPI below, to constructor of PlacementConnectorImpl.

Reply:
Ah, I didn't read your another comment. Now I know, it's done similarly to Experiences.